benefits/eligibility/views.py

```diff
@@ -217,16 +217,17 @@ def unverified(request):
     # tel: link to agency phone number
     agency = session.agency(request)
     buttons = viewmodels.Button.agency_contact_links(agency)
+    buttons.append(viewmodels.Button.home(request, _("enrollment.buttons.retry")))
```

Comment:
Can we please rename this `enrollment.buttons.retry` text to `core.buttons.retry` since it is now used in multiple places?

Reply:
Done